Ports/SuperTuxKart/package.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=SuperTuxKart
+useconfigure="false"
+version="1.4"
+archive_hash="9890392419baf4715313f14d5ad60746f276eed36eb580636caf44e2532c0f03"
+files="https://github.com/supertuxkart/stk-code/releases/download/${version}/supertuxkart-${version}-src.tar.xz ${port}-${version}-src.tar.xz $archive_hash"
+auth_type=sha256
+depends=("curl" "freetype" "glu" "harfbuzz" "libjpeg" "libopenal" "libogg" "libpng" "libvorbis" "openssl" "SDL2" "sqlite" "zlib")
```

Comment:
nit: one per line for better readability and to create nicer diffs when changing dependencies later on:

https://github.com/SerenityOS/serenity/blob/08c1effc04c08735621394b571da508fce91bd7b/Ports/python3/package.sh#L20-L30